### DISCUSSION TITLE
Mute on startup can't be disabled

### DISCUSSION THREAD

**Discussion by casabond** [SaifAqqad/AHK_MicMute#100] [Q&A] [answered]
Even though I have tried reactivating and disabling it several times, the microphone always remains muted when I start Windows, help me please

**Comment by SaifAqqad** [comment]
Does it still happen when Start on boot is disabled (i.e. when MicMute is not used)?

if so, it might be caused by the microphone itself setting a default state when windows starts up.

**Reply by casabond** [reply]
It also happens when micmute is not running, but it didn't happen before it was installed. Could it be due to the modification of some registry key?

**Reply by casabond** [reply, marked as answer]
I found the source of the problem: Microsoft powertoys, which I had installed the same day. So sorry for this thread. For anyone who has the same problem: there is the "start action" option at the bottom of the "deactivate video conference" module, which also works when the module is deactivated
